agents1/OfficialAgent.py

```diff
@@ -787,24 +787,12 @@ def _loadBelief(self, members, folder):
                     name = row[0]
                     competence = float(row[1])
                     willingness = float(row[2])
-                    # For each label we store the probability
-                    very_untrusted = float(row[3])
-                    untrusted = float(row[4])
-                    neutral = float(row[5])
-                    trusted = float(row[6])
-                    very_trusted = float(row[7])
-                    trustBeliefs[name] = TrustBelief(competence, willingness, very_untrusted, untrusted, neutral, trusted, very_trusted)
+                    trustBeliefs[name] = TrustBelief(competence, willingness)
                 # Initialize default trust values
-                if row and row[0]!=self._humanName:
+                if row and row[0] != self._humanName:
                     competence = default
                     willingness = default
-                    very_untrusted = float(0)
-                    untrusted = float(0)
-                    neutral = float(1)
-                    trusted = float(0)
-                    very_trusted = float(0)
-                    # TODO find good initial probabilities based on evaluation
-                    trustBeliefs[self._humanName] = TrustBelief(competence, willingness, very_untrusted, untrusted, neutral, trusted, very_trusted)
```

Comment:
Note we are not doing the probabilities over labels for now as it would make the decision making process and updating of values very complex and we want to start off with a simple model